### PR TITLE
Set version-specific rocm-dev flag

### PR DESCRIPTION
**Pull request by amdkila**
By default, this installs rocm-dev as expected. If the user passes the -v flag, or the --rocm-dev flag, the user can specify a point release version of rocm-dev. I.e. "-v rocm-dev-3.1" or something like that

### CODE REVIEW

**Comment by amdkila** [comment]
@pruthvistony can you verify that the package naming for the point release is ok? If the user passes the -v/--rocm-dev flag, the package that is installed is "rocm-dev-" then whatever the user passes into the flag, like 3.1 for example

File: install.sh [hunk 1]
```diff
@@ -155,10 +156,21 @@ install_packages( )
...
+      library_dependencies_fedora+=( "rocm-dev" )
+      library_dependencies_sles+=( "rocm-dev" )
+
+    else
+    # Install rocm-specific rocm-dev package
+      library_dependencies_ubuntu+=( "rocm-dev-${custom_rocm_dev}" )
+      library_dependencies_centos+=( "rocm-dev-${custom_rocm_dev}" )
+      library_dependencies_fedora+=( "rocm-dev-${custom_rocm_dev}" )
+      library_dependencies_sles+=( "rocm-dev-${custom_rocm_dev}" )
+    fi
```
Comment (leekillough) [inline install.sh]:
```
( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}"  )
```
to get rid of the `if` statement.

Comment (leekillough) [reply install.sh]: i.e.:
```
    # Install rocm-specific rocm-dev package
      library_dependencies_ubuntu+=( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}" )
      library_dependencies_centos+=( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}" )
      library_dependencies_fedora+=( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}" )
      library_dependencies_sles+=( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}" )
```

Comment (leekillough) [reply install.sh]: (slight correction edit made)

Comment (leekillough) [reply install.sh]: Sorry, I tried it several ways, and I can't seem to get it to work both set and unset.

**Comment by amdkila** [comment]
I'm going to wait for @pruthvistony to let me know what the name of the point release rocm-dev. IMO it should be rocm-dev-<ROCm_VERSION>

**Comment by amdkila** [comment]
> I'm going to wait for @pruthvistony to let me know what the name of the point release rocm-dev before merging. IMO it should be rocm-dev-<ROCm_VERSION>

Actually, should just force the full package name to be entered. Thanks @TorreZuk. I'll check CI logs and merge this right away, then cherry-pick it into master-rocm-3.2

**Pull request merged**